src/components/TitleComponent/TitleComponent.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import { Helmet } from "react-helmet";
+
+import { analytics } from "../../firebase";
+
+const defaultTitle = process.env.REACT_APP_TITLE;
+let currentPageId = null; //global state
+
+const TitleComponent = ({ title, pageId }) => {
+  const [analyticsScreen, setAnalyticsScreen] = useState(pageId);
+
+  useEffect(() => {
+    if (pageId !== currentPageId) {
+      currentPageId = pageId;
+      analytics.setCurrentScreen(currentPageId);
+    } else {
+      setAnalyticsScreen(defaultTitle);
```

Comment:
Why do you need this?

Reply:
Bem visto - eu fiz inicialmente so baseado no title, depois introduzi o pageId e nao ficou bem organizado. Fixing